terraform/modules/certificates/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "domain_name" {
+  type        = string
+  description = "Then domain name for which the certificate should be issued"
```

Comment:
I think it would be better to make this module create all the necessary certs for a given environment, rather than just be a wrapper for "make a certificate". Which means adjusting the inputs and naming to be sensible for that case.